nodes/src/nodes/router_llm/IGlobal.py

```
@@ -0,0 +1,79 @@
+# pipeline instances.
+# ------------------------------------------------------------------------------
+
+from rocketlib import IGlobalBase, OPEN_MODE
+from ai.common.config import Config
+
+
+class IGlobal(IGlobalBase):
+    # Holds the shared ModelRouter instance
+    router = None
+
+    def validateConfig(self, syntaxOnly: bool) -> None:
```

Ruff (0.15.7)

[warning] Boolean-typed positional argument in function definition

(FBT001)